File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Invs/INV6002U00GrdINV6002BeforeInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.INVS
{
    public class INV6002U00GrdINV6002BeforeInfo
    {
        private String _jaeryoCode;
        private String _jaeryoName;
        private String _subulDanuiName;
        private String _jaryoGubun;

        public String JaeryoCode
        {
            get { return this._jaeryoCode; }
            set { this._jaeryoCode = value; }
        }

        public String JaeryoName
        {
            get { return this._jaeryoName; }
            set { this._jaeryoName = value; }
        }

        public String SubulDanuiName
        {
            get { return this._subulDanuiName; }
            set { this._subulDanuiName = value; }
        }

        public String JaryoGubun
        {
            get { return this._jaryoGubun; }
            set { this._jaryoGubun = value; }
        }

        public INV6002U00GrdINV6002BeforeInfo() { }

        public INV6002U00GrdINV6002BeforeInfo(String jaeryoCode, String jaeryoName, String subulDanuiName, String jaryoGubun)
        {
            this._jaeryoCode = jaeryoCode;
            this._jaeryoName = jaeryoName;
            this._subulDanuiName = subulDanuiName;
            this._jaryoGubun = jaryoGubun;
        }

    }
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/invs/INV6002U00GrdINV6002BeforeInfo.java
package nta.med.data.model.ihis.invs;

public class INV6002U00GrdINV6002BeforeInfo {

	private String jaeryoCode;
	private String jaeryoName;
	private String subulDanuiName;
	private String jaryoGubun;

	public INV6002U00GrdINV6002BeforeInfo(String jaeryoCode, String jaeryoName, String subulDanuiName,
			String jaryoGubun) {
		super();
		this.jaeryoCode = jaeryoCode;
		this.jaeryoName = jaeryoName;
		this.subulDanuiName = subulDanuiName;
		this.jaryoGubun = jaryoGubun;
	}

	public String getJaeryoCode() {
		return jaeryoCode;
	}

	public void setJaeryoCode(String jaeryoCode) {
		this.jaeryoCode = jaeryoCode;
	}

	public String getJaeryoName() {
		return jaeryoName;
	}

	public void setJaeryoName(String jaeryoName) {
		this.jaeryoName = jaeryoName;
	}

	public String getSubulDanuiName() {
		return subulDanuiName;
	}

	public void setSubulDanuiName(String subulDanuiName) {
		this.subulDanuiName = subulDanuiName;
	}

	public String getJaryoGubun() {
		return jaryoGubun;
	}

	public void setJaryoGubun(String jaryoGubun) {
		this.jaryoGubun = jaryoGubun;
	}

}
